src/openrct2/interface/Viewport.h

```diff
@@ -17,11 +17,18 @@
 #ifndef _VIEWPORT_H_
 #define _VIEWPORT_H_
 
-#include "../world/Map.h"
-#include "../world/Sprite.h"
-#include "../paint/Paint.h"
 #include "Window.h"
 
+struct paint_session;
```

Comment:
Is this block really worth the reduced inclusions?

Reply:
Why would you not want to forward declare things when you can? My answer is: yes.

Reply:
I agree with @AaronVanGeffen here. One or two is fine, but I think this is too many.

Reply:
Map, Sprite and Paint are very big headers though, so this is most likely going to have the biggest impact on performance.

Reply:
Well, all right, I'll leave the decision to you then.

Reply:
It's super annoying making changes to anything in the peep or vehicle headers they get included everywhere. Anything that reduces that would be great